tests/core/pyspec/eth2spec/test/helpers/genesis.py

```diff
@@ -6,11 +6,12 @@
 
 
 def build_mock_validator(spec, i: int, balance: int):
-    pubkey = pubkeys[i]
+    active_pubkey = pubkeys[i]
+    withdrawal_pubkey = pubkeys[-1 - i]
```

Comment:
this is a bit weird but given the domain separation b/t active and withdrawal keys, it is prolly ok

best to just duplicate the `pubkeys` infra for withdrawal keys but this is a quick and dirty solution

Reply:
right -- I mainly didn't want active and withdrawal key (behind the hash) to be the same (as it was).

This at least ensures they are generally different from each other and uses the pre-generated (which takes time) keys in a deterministic way